src/index.html

```diff
@@ -11,7 +11,7 @@
     <meta http-equiv="pragma" content="no-cache" />
 
     <title>Alfresco Content App</title>
-    <base href="/" />
+    <base href="./" />
```

Comment:
This is not the right way of changing the base ref
Angular provides an option that can be used during the build time
`ng build --prod --base-href .`
So, in my opinion, the package.json build command should be changed.

Reply:
There is a script for that already in package.json called build.tomcat

Reply:
Yeah, this change will break all "normal", non-Tomcat deployments